src/main/resources/theme-resources/messages/messages_en.properties

```diff
@@ -1,10 +1,9 @@
-authCodeText=Your one-time code is %1$s and is valid for %2$d minutes.
-
-smsAuthTitle=SMS Code
-smsAuthCodeTitle=SMS Code
-smsAuthLabel=Code
-smsAuthInstruction=Enter the code we sent to your device.
+authCodeText=%1$s is your one time passcode from OHCRN. %1$s est votre code d'acc�s � usage unique de l'OHCRN.
```

Comment:
Should this message be only in english?

Reply:
Ah I should have called that out in my PR description - we're providing both languages in the text verbiage (and emails) because we're not (yet) capturing the user's preferred language as part of their account info